src/reconstruct_loopset.jl

```diff
@@ -378,7 +378,7 @@ function avx_loopset(instr, ops, arf, AM, LPSYM, LB, vargs)
     resize!(ls.loop_order, ls.loopsymbol_offsets[end])
     arraysymbolinds = gen_array_syminds(AM)
     opsymbols = [gensym(:op) for _ ∈ eachindex(ops)]
-    nopsv = calcnops.(Ref(ls), ops)
+    nopsv = [calcnops(ls, op) for op in ops]
```

Comment:
Some time ago I found that compilation time was shorter for `map` than broadcasting, and I presume that extends to comprehensions. But this is purely cosmetic.

Reply:
Good call. I'll stick with comprehensions or maps.